src/AvTranscoder/filter/FilterGraph.cpp

```diff
@@ -275,7 +275,7 @@ void FilterGraph::process(const std::vector<IFrame*>& inputs, IFrame& output)
     {
         // Retrieve frame from buffer or directly from input
         IFrame* inputFrame = (bypassBuffers)? inputs.at(index) : _inputAudioFrameBuffers.at(index).getFrameSampleNb(minInputFrameSamplesNb);
-        const int ret = av_buffersrc_add_frame_flags(_filters.at(index)->getAVFilterContext(), &inputFrame->getAVFrame(), AV_BUFFERSRC_FLAG_PUSH & AV_BUFFERSRC_FLAG_KEEP_REF);
```

Comment:
so it's `av_buffersrc_add_frame` https://ffmpeg.org/doxygen/trunk/group__lavfi__buffersrc.html#ga8fc71cb48c1ad1aa78b48f87daa4cf19 as it's equivalent ...

Reply:
it's without not with .. I close the remark